saleor/order/forms.py

```diff
@@ -49,11 +49,11 @@ def __init__(self, *args, **kwargs):
         self.fields['email'].widget = forms.HiddenInput()
 
 
-class OrderNoteForm(forms.ModelForm):
+class CustomerNoteForm(forms.ModelForm):
+    customer_note = forms.CharField(
+        max_length=250, required=False, strip=True, label=False,
```

Comment:
Aren't we missing a label here?

Reply:
There was no label in note form during checkout so I didn't add it here as well. I can add a label or simply a header like "Add a note to Your order" or leave it as it is. How would You advice is better?

Reply:
The current one was "Add note to order", it seems good enough.

Reply:
I've added a header to the template